src/components/announcements-listing/AnnouncementsList.tsx

```diff
@@ -0,0 +1,169 @@
+  color: #bfbfbf;
+`;
+
+const noAnnouncementsTexts = (
+  <>
+    <NoAnnouncementsText>{NO_ANNOUNCEMENTS_HEADER}</NoAnnouncementsText>
+    <NoAnnouncementsSubText>
+      {NO_ANNOUNCEMENTS_SUBHEADER}
+    </NoAnnouncementsSubText>
+  </>
+);
+
+export interface AnnouncementsListProps {
+  announcements: Announcement[];
+}
+
+const AnnouncementsList: React.FC<AnnouncementsListProps> = ({
+  announcements,
+}) => {
+  const [announcementsPerPage, setAnnouncementsPerPage] = useState<number>(0);
+  const [currentPage, setCurrentPage] = useState<number>(1);
+
+  const handleNextPage = (event: any) => {
+    setCurrentPage(currentPage + 1);
+  };
+
+  const handlePreviousPage = (event: any) => {
+    setCurrentPage(currentPage - 1);
+  };
+
+  const handleNoOnClick = (event: any) => {
+    setCurrentPage(currentPage);
+  };
+
+  const handlePageClick = (event: any) => {
+    setCurrentPage(event.target.value);
+  };
+
+  const noAnnouncements = announcements.length === 0;
+  const indexOfLastAnnouncement = currentPage * announcementsPerPage;
+  const indexOfFirstAnnouncement =
+    indexOfLastAnnouncement - announcementsPerPage;
+  const currentAnnouncements = announcements.slice(
+    indexOfFirstAnnouncement,
+    indexOfLastAnnouncement,
+  );
+
+  const pageNumbers = [];
+  for (
+    let i = 1;
+    i <= Math.ceil(announcements.length / announcementsPerPage);
+    i++
+  ) {
+    pageNumbers.push(i);
+  }
+
+  useEffect(() => {
+    setAnnouncementsPerPage(6);
+  }, []);
+
+  const renderPageNumbers = pageNumbers.map((num: number) => {
+    if (num !== currentPage) {
+      return (
+        <PageNumber key={num} value={num} onClick={handlePageClick}>
+          {num}
+        </PageNumber>
+      );
+    } else {
+      return (
+        <SelectedPageNumber key={num} value={num}>
+          {num}
+        </SelectedPageNumber>
+      );
+    }
+  });
+
+  return (
+    <>
+      {noAnnouncements ? (
+        noAnnouncementsTexts
+      ) : (
+        <>
+          <AnnouncementsListWrapper>
+            {currentAnnouncements.map((announcement, i) => {
+              return <AnnouncementCard {...announcement} key={i} />;
+            })}
+          </AnnouncementsListWrapper>
+
+          {announcements.length > announcementsPerPage ? (
+            <PageNumbersWrapper>
+              <ArrowButton
+                key="prev"
+                onClick={
+                  currentPage === 1 ? handleNoOnClick : handlePreviousPage
+                }
+              >
+                {'<'}
+              </ArrowButton>
+              {renderPageNumbers}
+              <ArrowButton
+                key="next"
+                onClick={
+                  currentPage === pageNumbers[pageNumbers.length - 1]
+                    ? handleNoOnClick
+                    : handleNextPage
+                }
+              >
+                {'>'}
+              </ArrowButton>
+            </PageNumbersWrapper>
+          ) : null}
```

Comment:
Instead of doing: 
`condition ? <render html> : null`

you could just do:
`condition && <render html>`